File: Prog52aLangs/Prog52a.c
#include <math.h>
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>

void main() {
    printf("Enter the length of your rectangle: \n");
    int length; scanf("%d", &length);
    printf("Enter the width of your rectangle: \n");
    int width; scanf("%d", &width);
    int area = length * width;
    int perim = 2*length + 2*width;
    printf("Area: %d\n", area);
    printf("Perim: %d\n", perim);
}

File: Prog52aLangs/Prog52a.rs
#[macro_use]
mod cin;
use std::io::{Write};

fn main() {
    println!("Enter width: ");
    input!(width as i32);
    println!("Enter length: ");
    input!(length as i32);

    let area = width * length;
    let perim = 2*width + 2*length;
    println!("Area: {}", area);
    println!("Perim: {}", perim);
}